ui/component/walletSendTip/view.jsx

```diff
@@ -262,8 +273,8 @@ function WalletSendTip(props: Props) {
           Lbryio.call(
```

Comment:
We should try to move this component to /web at least. Ideally This is a function imported from lbryinc repo.